src/Makefile

```diff
@@ -24,7 +30,7 @@ else
 endif
 SHOBJ_LDFLAGS += $(LDFLAGS)
 
-export CFLAGS
+# export CFLAGS
```

Comment:
Why did you comment out this export?

Reply:
It's better not to use CFLAGS at all, as it functions as a backdoor with implicit rules. Disabling implicit rules, setting compilation flags explicitly, and using Makefile includes for sharing code is the way to go.